setup.py

```diff
@@ -20,59 +20,4 @@
 
 from setuptools import setup
 
-# grab metadata without importing the module
-metadata = {}
-with open("pdudaemon/__about__.py", encoding="utf-8") as fp:
-    exec(fp.read(), metadata)
-
-# Setup the package
-setup(
-    name='pdudaemon',
-    version=metadata['__version__'],
-    description=metadata['__description__'],
-    author=metadata['__author__'],
-    author_email='matt@mattface.org',
-    license=metadata['__license__'],
-    url=metadata['__url__'],
-    python_requires=">=3.4",
-    classifiers=[
-        "Development Status :: 5 - Production/Stable",
-        "Environment :: Console",
-        "Intended Audience :: Developers",
-        "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
-        "Operating System :: POSIX :: Linux",
-        "Programming Language :: Python :: 3.4",
-        "Programming Language :: Python :: 3.5",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3 :: Only",
-        "Topic :: Communications",
-        "Topic :: Software Development :: Testing",
-        "Topic :: System :: Networking",
-    ],
-    packages=['pdudaemon', 'pdudaemon.drivers'],
-    entry_points={
-        'console_scripts': [
-            'pdudaemon = pdudaemon:main'
-        ]
-    },
-    install_requires=[
-        "aiohttp",
-        "requests",
-        "pexpect",
-        "systemd_python",
-        "paramiko",
-        "pyserial",
-        "hidapi",
-        "pysnmp",
-        "pyasn1",
-        "pyusb",
-        "pymodbus",
-    ],
-    extras_require={
-        'test': [
-            'pytest',
-            'pytest-mock',
-        ],
-    },
-    zip_safe=True
-)
+setup()
```

Comment:
We should be able to get rid of `setup.py` with a `[build-system]` section inside pyproject.toml, i.e.

```
[build-system]
requires = [
  "setuptools>=<version-requirement-for-pdudaemon"
]
build-backend = "setuptools.build_meta"
```

Reply:
Ah, I now realised that this switches over to `setup.cfg` instead of `pyproject.toml` we should probably completely switch to `pyproject.toml` instead of retaining `setup.cfg`.